llvm/lib/Transforms/IPO/RepoPruning.cpp

```diff
@@ -213,57 +274,31 @@ static bool doPruning(Module &M) {
     if (GO.isDeclaration() || GO.hasAvailableExternallyLinkage() ||
         !isSafeToPrune(GO))
       return false;
-    auto const Result = repodefinition::get(&GO);
-    assert(!Result.second &&
-           "The repo_definition metadata should be created by "
-           "the RepoMetadataGeneration pass!");
 
+    auto const Result = repodefinition::get(&GO);
```

Comment:
Since the result of function is a pair of the global object's digest value and a bool which shows whether GO contains the `RepoDefinition` metadata, I  renamed it from `Result` to `DigestStatus` .